EIPS/eip-5793.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 5793
+title: eth/68 - Add tx type to tx announcement
+description: Adds the transaction type and transaction size to tx announcement messages in the wire protocol
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: https://ethereum-magicians.org/t/eip-5793-eth-68-add-transaction-type-to-tx-announcement/11364
+status: Draft
+type: Standards Track
+category: Networking
+created: 2022-10-18
+requires: 2464, 2481, 4938
+---
+
+## Abstract
+
+The [Ethereum Wire Protocol](https://github.com/ethereum/devp2p/tree/master/caps/eth.md) defines request and response messages for exchanging data between clients. The `NewPooledTransactionHashes` message announces transactions available in the node. This EIP extends this announcement message such that beside the transaction hashes, the node sends the transaction types and their sizes (as defined in EIP-2718) as well.
```

Comment:
```suggestion
The Ethereum Wire Protocol defines request and response messages for exchanging data between clients. The `NewPooledTransactionHashes` message announces transactions available in the node. This EIP extends this announcement message such that beside the transaction hashes, the node sends the transaction types and their sizes (as defined in [EIP-2718](./eip-2718.md)) as well.
```
- **[Blocker]** Remove link to wire protocol
- **[Blocker]** Add link to EIP-2718